src/common/MuxConfig.h

```diff
@@ -316,6 +316,24 @@ class MuxConfig
      */
     inline bool getIfEnableDefaultRouteFeature() {return mEnableDefaultRouteFeature;};
 
+    /**
+     * @method getUseKnownMacActiveActive
+     * 
+     * @brief check if use known mac to probe for active-active ports
+     * 
+     * @return true to use known mac to probe for active-active ports
+     */
+    inline bool getUseKnownMacActiveActive() { return mUseKnownMacActiveActive; };
+
+    /**
+     * @method setUseKnownMacActiveActive
+     * 
+     * @brief setter to set if use known mac to probe for active-active ports
+     * 
+     * @return none
+     */
+    inline void setUseKnownMacActiveActive(bool useKnownMacActiveActive) { mUseKnownMacActiveActive = useKnownMacActiveActive; };
```

Comment:
If config is updated to `false`, will previously updated server mac be consumed? No?

Reply:
Now mux port config will cache the last mac-address update from Netlink, and use that if `useKnownMac` is set to `false`